samples/createTableRangePartitioned.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(datasetId = 'my_dataset', tableId = 'my_table') {
+  // [START bigquery_create_table_range_partitioned]
+  // Import the Google Cloud client library
+  const {BigQuery} = require('@google-cloud/bigquery');
+  const bigquery = new BigQuery();
+
+  async function createTableRangePartitioned() {
+    // Creates a new integer range partitioned table named "my_table"
+    // in "my_dataset".
+
+    /**
+     * TODO(developer): Uncomment the following lines before running the sample.
+     */
+    // const datasetId = "my_dataset";
+    // const tableId = "my_table";
+
+    const schema = [
+      {name: 'fullName', type: 'STRING'},
+      {name: 'city', type: 'STRING'},
+      {name: 'zipcode', type: 'INTEGER'},
+    ];
+
+    // To use integer range partitioning, select a top-level REQUIRED or
+    // NULLABLE column with INTEGER / INT64 data type.
```

Comment:
nit:  should we indicate what happens to values outside the range?  Or is there a link in the docs we can send people to get more information here?